src/jshint.js

```diff
@@ -4584,7 +4605,25 @@ var JSHINT = (function() {
   FutureReservedWord("public", { es5: true, strictOnly: true });
   FutureReservedWord("short");
   FutureReservedWord("static", { es5: true, strictOnly: true });
-  FutureReservedWord("super", { es5: true });
+  FutureReservedWord("super", {
+    es5: true,
+    nud: function() {
+      if (state.inESNext() && checkPunctuators(state.tokens.next, "(")) {
+        var f = state.funct;
+        while (f) {
+          // If `super` is called in an arrow function, check the outer scope
+          if (!f["(arrow)"]) { break; }
+          f = f["(context)"];
+        }
+        if (!f["(super)"]) {
+          error("E061", state.tokens.curr);
+        } else {
+          f["(super)"] = "called";
```

Comment:
is it okay to call super twice? again not a regression so we could handle in a different pr